templates/user-settings.php

```diff
@@ -134,6 +134,26 @@ class="<?php echo \esc_attr( $classes_for_button ); ?>"
 				</p>
 			</td>
 		</tr>
+		<tr scope="row">
+			<th>
+				<label><?php \esc_html_e( 'Account Aliases', 'activitypub' ); ?></label>
```

Comment:
There may be an opportunity to add this to the glossary, unless we're happy with the textarea description.